.github/workflows/daily-wpt-fyi.yml

```diff
@@ -96,16 +96,18 @@ jobs:
           if-no-files-found: warn
       - name: Upload WPT Report to wpt.fyi API
         env:
-          WPT_FYI_ENDPOINT: ${{ vars.WPT_FYI_ENDPOINT }}
           WPT_FYI_USERNAME: ${{ vars.WPT_FYI_USERNAME }}
           WPT_FYI_PASSWORD: ${{ secrets.WPT_FYI_PASSWORD }}
         run: |
           if [ -e out/wpt/wptreport.json ]; then
             cd out/wpt
             gzip wptreport.json
-            curl \
-              -u "$WPT_FYI_USERNAME:$WPT_FYI_PASSWORD" \
-              -F "result_file=@wptreport.json.gz" \
-              -F "labels=master" \
-              $WPT_FYI_ENDPOINT
+            for WPT_FYI_ENDPOINT in "https://wpt.fyi/api/results/upload" "https://staging.wpt.fyi/api/results/upload"
```

Comment:
Does this loop ignore errors? If it does not, perhaps we should try the upload to staging first?

Reply:
Exactly because it doesn't ignore errors I've put production first, so that if staging is not available (the nature of a staging env) the more important service gets the report.

Reply:
Okay fair enough.